src/editor/EditorCommandHandlers.js

```diff
@@ -1142,26 +1148,53 @@ define(function (require, exports, module) {
 
         return result.promise();
     }
+<<<<<<< HEAD
```

Comment:
this is a file with merge conflict code in. Please fix your commits first, and verify that your `npm test` *and* `npm run build` work. There is no way this file works.